src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -656,20 +661,25 @@ public void TestMinWorkingSet()
             }
         }
 
+        // XXX No longer skip: OSX, FreeBSD
         [Fact]
-        [SkipOnPlatform(TestPlatforms.OSX | TestPlatforms.FreeBSD | TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS, "Getting MinWorkingSet is not supported on OSX, BSD, iOS, MacCatalyst, and tvOS.")]
+        [SkipOnPlatform(TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS, "Getting MinWorkingSet is not supported on OSX, BSD, iOS, MacCatalyst, and tvOS.")]
         public void MinWorkingSet_GetNotStarted_ThrowsInvalidOperationException()
         {
             var process = new Process();
             Assert.Throws<InvalidOperationException>(() => process.MinWorkingSet);
         }
 
+        // The correct exception for a new process is: InvalidOperationException
+        // With this fix: https://github.com/dotnet/runtime/issues/105422
+        // this test could be replaced by the previous one.  However, on
+        // MacOS the above is skipped and this is run.  XXX: Why? -gwr
         [Fact]
         [PlatformSpecific(TestPlatforms.OSX | TestPlatforms.FreeBSD)]
         public void MinWorkingSet_GetMacos_ThrowsPlatformSupportedException()
         {
             var process = new Process();
-            Assert.Throws<PlatformNotSupportedException>(() => process.MinWorkingSet);
+            Assert.Throws<InvalidOperationException>(() => process.MinWorkingSet);
```

Comment:
This makes it a duplicate of `MinWorkingSet_GetNotStarted_ThrowsInvalidOperationException`?